src/main/java/mate/academy/spring/model/Role.java

```diff
@@ -0,0 +1,40 @@
+package mate.academy.spring.model;
+
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "roles")
+public class Role {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Enumerated(value = EnumType.STRING)
+    private RoleName roleName;
+
+    public enum RoleName {
+        ROLE_ADMIN,
+        ROLE_USER
```

Comment:
```suggestion
        ADMIN,
        USER
```
prefix should be added in the class where I added comment about this prefix